### ISSUE TITLE
conda-forge package

### ISSUE THREAD

**Issue by davidbrochart** [erdc/proteus#1018]
I'm trying to build Proteus for conda-forge: https://github.com/conda-forge/staged-recipes/pull/9942
I am running into several problems. To start with, do you recommend taking the latest release of Proteus (1.7.0)?

**Comment by davidbrochart** [comment]
It seems that Chrono is a dependency of Proteus (1.7.0), can you please confirm @cekees?

**Comment by cekees** [comment]
That's right. We can disable the modules that depend on it if it becomes a problem. It's got a pretty straightforward build system. The hashdist recipe may be useful:

https://github.com/erdc/stack/blob/master/pkgs/chrono/chrono.yaml

**Comment by cekees** [comment]
chrono support updated to https://github.com/projectchrono/chrono/releases#release-10.0.0
